ui/apps/platform/src/Containers/Collections/CollectionForm.css

```diff
@@ -0,0 +1,17 @@
+.collection-form-expandable-section {
+  background-color: var(--pf-global--BackgroundColor--100);
+  padding: var(	--pf-global--spacer--md);
+}
+
+.collection-form-expandable-section button.pf-c-expandable-section__toggle {
+  align-items: center;
+  color: inherit;
+  font-weight: inherit;
+  font-size: inherit;
+  text-align: left;
+}
+
+.collection-form-expandable-section button.pf-c-expandable-section__toggle:focus {
+  /* This blue outline on focus is a global problem, but it is especially jarring in this case */
+  outline: none;
+}
```

Comment:
Aha, a gap in lint configuration not to report that file does not end with newline.

Brad has experience and interest in lint, so maybe could delegate for him to follow up on config?